crates/router/src/connector/globalpay/requests.rs

```diff
@@ -167,18 +183,32 @@ pub struct Notifications {
 }
 
 #[derive(Debug, Serialize, Deserialize)]
+
 pub struct Order {
     /// Merchant defined field common to all transactions that are part of the same order.
     pub reference: Option<String>,
 }
 
-#[derive(Debug, Default, Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub enum PaymentMethodData {
+    Card(Card),
+    Apm(Apm),
+    BankTransfer(BankTransfer),
+    DigitalWallet(DigitalWallet),
+}
+
+impl Default for PaymentMethodData {
```

Comment:
Why default is required?

Reply:
the `#[default]` attribute may only be used on unit enum variants
. Manual implementation of `Default` should be done in this case

Reply:
For `PaymentMethodData` how can we pass a default value in the request, for this we can't pass default values.

Reply:
I have removed default for `PaymentMethodData`